src/features/enrollments/MemberEnrollmentList.jsx

```diff
@@ -28,17 +30,32 @@ export default function MemberEnrollmentList({ offerings }) {
   const data = members.map(member => ({
     name: member.name,
     email: member.email,
+    cohort: member.cohort,
```

Comment:
I think this may be a problem when a user is a member of more than one cohort within the same partnership.

Reply:
Ah, right - so this should be a list then, and the cohort filter should search for a matching value in that list I think.